_data/simple-icons.json

```diff
@@ -13978,6 +13978,11 @@
             "source": "https://brand.scribd.com/d/oFZcsq7FVpSh/about-scribd#/media-press-kit/downloads",
             "guidelines": "https://brand.scribd.com/d/oFZcsq7FVpSh/about-scribd#/marketing-visual-guidelines/our-logo/our-logomark"
         },
+        {
+            "title": "Scribus",
+            "hex": "133B6B",
```

Comment:
Because it seemed to be on of the more dominant colors on their website. See
![image](https://github.com/simple-icons/simple-icons/assets/19874562/5b09e2bf-0512-4289-8b79-857a1a99babf)

Reply:
Fair enough